test/components/scannerversion.test.js

```diff
@@ -48,7 +48,7 @@ describe('Scanner Node Software Version', function () {
     describe('upgrade', async function () {
         it('sets version', async function () {
             const ScannerVersion_0_1_0 = await ethers.getContractFactory('ScannerNodeVersion_0_1_0');
-            const originalScannerVersion = await upgrades.deployProxy(ScannerVersion_0_1_0, [this.contracts.access.address, this.contracts.router.address], {
+            const originalScannerVersion = await upgrades.deployProxy(ScannerVersion_0_1_0, [this.contracts.access.address, this.contracts.access.address], {
```

Comment:
We shouldn't be using a proper contract in place of the router. I would prefer to use an empty contract as a mock router. Same applies to the other test files.

Reply:
Fair enough e2f1d36